### PR TITLE
[CrowdStrikeFalcon] Behaviours Pattern Disposition Details (new field)

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

image
image

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by efelmandar — COMMENTED** [review]
@ennozdd Thank you very much for your contribution it is much appreciated, I finished reviewing your changes and everything looks great! well done! 
@RotemAmit the PR is approved 👍

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#### Mappers
+
+##### CrowdStrike Falcon Mapper
+
+- Mapped the new field **Behaviour Pattern Disposition Details**.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md:6]:
```suggestion
Mapped the new field **Behaviour Pattern Disposition Details**.
```

Is that how you are spelling Behaviour?  It should be Behavior.  With the u is a British spelling.

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+#### Incident Fields
+
+- New: **Behaviour Pattern Disposition Details**.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md:11]:
```suggestion
New: **Behaviour Pattern Disposition Details**.
```

Same spelling comment re: Behaviour.

**Comment by ShirleyDenkberg** [comment]
@RotemAmit @efelmandar Doc review completed.

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md (hunk 1 above)
Comment (ennozdd) [reply Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_24.md:6]: other fields in the content pack are spelled *behaviour*, I thought I should follow the convention.

**Comment by efelmandar** [comment]
Hi @ennozdd I see that some of the content validations are failing and are critical for merging the PR, please run the following command `demisto-sdk format -i Packs/CrowdStrikeFalcon/IncidentFields/incidentfield-Behaviour_Pattern_Disposition_Details.json`, which will format the incident field file and should solve the errors.

besides that, I think there is some conflict with the release notes, so please try to update release notes as well, using the following command, `demisto-sdk update-release-notes -i Packs/CrowdStrikeFalcon`

**Comment by ennozdd** [comment]
@efelmandar done

```[2023-06-21 09:40:21,023] [INFO] /home/circleci/project/Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json: [GR103] - Content item 'CrowdStrike Falcon Mapper' using content items: Behavior which cannot be found in the repository.```

Validation fails but I haven't changed anything related to `Behavior` field

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+#### Mappers
+
+##### CrowdStrike Falcon Mapper
+
+- Mapped the new field **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
+
+
+#### Incident Fields
+
+- New: **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
```
Comment (RotemAmit) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md:11]:
```suggestion
New incident field: 
- **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
```

**Review by RotemAmit — COMMENTED** [review]
Thank you very much for your contribution!

**Comment by RotemAmit** [comment]
Hi @ennozdd,

Again, thank you very much for your contribution!
Can you send us a short video with your changes, so we will be able to see that everything is working as expected?
You can send it to my email, here or via DFIR.

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+##### CrowdStrike Falcon Mapper
+
+- Mapped the new field **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
+
+
+#### Incident Fields
+
+New incident field: 
+- **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
```
Comment (RotemAmit) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md:12]:
```suggestion
#### Incident Fields
- **CrowdStrike Falcon Behaviour Pattern Disposition Details** - new incident field.
```

**Review by RotemAmit — COMMENTED** [review]
Please resolve the conflicts with master by bumping the version of the pack in the pack_metadata.json file and rename the release notes file to 1_10_28.md.

**Comment by efelmandar** [comment]
@ennozdd Thank you again for your contribution! I've watched the recorded demo and it looks good, nice work!

@RotemAmit approved

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_30.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+#### Mappers
+
+##### CrowdStrike Falcon Mapper
+
+- Mapped the new field **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
+
+
+#### Incident Fields
+
+- New: **CrowdStrike Falcon Behaviour Pattern Disposition Details**.
```
Comment (RotemAmit) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_30.md:11]:
```suggestion
#### Incident Fields
##### CrowdStrike Falcon Behaviour Pattern Disposition Details
- New incident field.
```
Please update the release notes as in the suggestion.

**Review by RotemAmit — APPROVED** [review]

**Pull request merged**